modules/graceful-shutdown.adoc

```diff
@@ -21,6 +21,14 @@ You can shut down a cluster until a year from the installation date and expect i
 [IMPORTANT]
 ====
 It is important to take an etcd backup before performing this procedure so that your cluster can be restored if you encounter any issues when restarting the cluster.
+
+Even though the cluster is expected to be functional after the restart, it might not recover due to unexpected conditions, such as:
```

Comment:
This sentence ("Even though...") is hard to parse and might be misleading. The ISG also states "To avoid possible translation problems, use complete sentences to introduce lists and not phrases." Suggested rephrase:

`For example, the following conditions can cause the restarted cluster to malfunction:`

Reply:
Addressed